src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -265,6 +426,42 @@ public function iShouldSeeTwoCancelledPaymentsAndNewOneReadyToBePaid()
     }
 
     /**
+     * @Then I should be notified that the order has been successfully addressed
+     */
+    public function iShouldBeNotifiedThatTheOrderHasBeenSuccessfullyAddressed()
```

Comment:
Haha, we should disable the flash. That's annoying for every checkout step. So this step should go away. ;)

```
_sylius:
    flash: false
```